src/ci/docker/host-x86_64/dist-x86_64-linux/build-gccjit.sh

```diff
@@ -1,19 +1,22 @@
 #!/usr/bin/env bash
 
+GIT_REPO="https://github.com/antoyo/gcc"
+
+# This commit hash needs to be updated to use a more recent gcc fork version.
+GIT_COMMIT="78dc50f0e50e6cd1433149520bd512a4e0eaa1bc"
+
 set -ex
 
 cd $1
 
 source shared.sh
 
 # Setting up folders for GCC
-git clone https://github.com/antoyo/gcc gcc-src
-cd gcc-src
-# This commit hash needs to be updated to use a more recent gcc fork version.
-git checkout 78dc50f0e50e6cd1433149520bd512a4e0eaa1bc
+curl -L "$GIT_REPO/archive/$GIT_COMMIT.tar.gz" |
+    tar -xz --transform "s/gcc-$GIT_COMMIT/gcc-src/"
 
-mkdir ../gcc-build ../gcc-install
-cd ../gcc-build
+mkdir gcc-build gcc-install
+cd gcc-build
```

Comment:
Should we use [`pushd`&`popd`](https://en.wikipedia.org/wiki/Pushd_and_popd) ?
```suggestion
pushd gcc-build
```

Reply:
I'm okay with that, but we have a pretty inconsistent mix of both already in various scripts.